test/e2e/gke_cluster_test.go

```diff
@@ -60,7 +60,7 @@ func testGKECluster(t *testing.T, createSubnet bool) {
 
 	t.Logf("configuring the GKE cluster PROJECT=(%s) LOCATION=(%s)", gkeProject, gkeLocation)
 	builder := gke.NewBuilder([]byte(gkeCreds), gkeProject, gkeLocation)
-	builder.WithClusterMinorVersion(1, 23)
+	builder.WithClusterMinorVersion(1, 24)
```

Comment:
This should be a separate change IMHO.

Reply:
Separate PR https://github.com/Kong/kubernetes-testing-framework/pull/801

Reply:
I wonder why it doesn't show a conflict given that #801 got already merged 🤔